source/views/sis/student-work/types.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+export type JobType = {
+  id: number,
+  comments: string,
+  contactEmail: string,
+  contactFirstName: string,
+  contactLastName: string,
+  contactPhone: number,
+  description: string,
+  hoursPerWeek: string,
+  lastModified: string,
+  office: string,
+  skills: string,
+  timeOfHours: any,
```

Comment:
In general, you almost never want to annotate with `any`. You should either use a narrower type (like "string | number"), or a generic type if you're doing something like Array#map (`map = (Array<T>) => Array<U>` or whatever). Almost never `any` itself.